test/bin/configure_hypervisor_firewall.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# This script should be run on the hypervisor to configure the
+# firewall for incoming connections to the VM network.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+VM_BRIDGE=$(sudo virsh net-info default | grep '^Bridge:' | awk '{print $2}')
+VM_BRIDGE_CIDR=$(ip -f inet addr show "${VM_BRIDGE}" | grep inet | awk '{print $2}')
+sudo firewall-cmd --permanent --zone=trusted --add-source="${VM_BRIDGE_CIDR}"
```

Comment:
Why do we need this if we're explicitly opening port 8080 in the public zone? What other connections over the trusted network do we expect?

Reply:
I'm not entirely sure. I was not able to get the VMs to be able to see the server without this change. Maybe because by default the VMs cannot talk to the hypervisor?